src/EditorFeatures/CSharp/BlockCommentEditing/BlockCommentEditingCommandHandler.cs

```diff
@@ -14,9 +14,11 @@
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.BlockCommentEditing
 {
-    [ExportCommandHandler(nameof(BlockCommentEditingCommandHandler), ContentTypeNames.CSharpContentType)]
+    [Export(typeof(VisualStudio.Commanding.ICommandHandler))]
+    [ContentType(ContentTypeNames.CSharpContentType)]
+    [Name(nameof(BlockCommentEditingCommandHandler))]
     [Order(After = PredefinedCommandHandlerNames.Completion)]
-    internal class BlockCommentEditingCommandHandler : AbstractBlockCommentEditingCommandHandler
```

Comment:
AbstractBlockCommentEditingCommandHandler is used by TypeScript so I cannot change it. So I refactored it into BaseAbstractBlockCommentEditingCommandHandler and PlatformAbstractBlockCommentEditingCommandHandler (not happy about that name though). Once TypeScript migrates to the new commanding, this can be simplified back.